### PR TITLE
Adds two more IMPORANT color burger colors

### PR DESCRIPTION
**Pull request by jughu**
:cl: Improvedname
add: Adds racial equality to color burgers
/:cl:

[why]: # (Please add a short description [two lines down] of why you think these changes would benefit the game. If you can't justify it in words, it might not be worth adding.) WE HAVE THESE CRAYONS SO WHY NOT THESE COLOURS

### CODE REVIEW

**Review by Cyberboss — DISMISSED** [review]

**Comment by lordpidey** [comment]
They could use better descriptions.  You just copy-pasted the purple one, which doesn't make sense.

**Comment by Iamgoofball** [comment]
Missing the point of the Pretty Patties

**Review by Cyberboss — CHANGES_REQUESTED** [review]
Add unique descriptions for them

**Comment by ShadowDeath6** [comment]
catburgers when?

**Comment by jughu** [comment]
@Cyberboss hows the black one i feel good with the white

File: code/modules/food_and_drinks/food/snacks_burgers.dm [hunk 1]
```diff
@@ -176,6 +176,22 @@
 	color = "#DA00FFFF"
 	bonus_reagents = list("purplecrayonpowder" = 10, "vitamin" = 5)
 	foodtype = GRAIN | MEAT
+	
+/obj/item/reagent_containers/food/snacks/burger/black
+	name = "black burger"
+	desc = "This is overcooked"
```
Comment (nicbn) [inline code/modules/food_and_drinks/food/snacks_burgers.dm]: This doesn't have a dot

File: code/modules/food_and_drinks/food/snacks_burgers.dm [hunk 2]
```diff
@@ -176,6 +176,22 @@
...
+	icon_state = "cburger"
+	color = "#1C1C1C"
+	bonus_reagents = list("blackcrayonpowder" = 10, "vitamin" = 5)
+	foodtype = GRAIN | MEAT
+	
+/obj/item/reagent_containers/food/snacks/burger/white
+	name = "white burger"
+	desc = "How lifeless."
```
Comment (nicbn) [inline code/modules/food_and_drinks/food/snacks_burgers.dm]: This does have a dot

File: code/modules/food_and_drinks/food/snacks_burgers.dm (hunk 1 above)
Comment (Supermichael777) [reply code/modules/food_and_drinks/food/snacks_burgers.dm]: period

File: code/modules/food_and_drinks/food/snacks_burgers.dm (hunk 2 above)
Comment (Supermichael777) [reply code/modules/food_and_drinks/food/snacks_burgers.dm]: i would have gone with Delicious titanium

**Review by Cyberboss — CHANGES_REQUESTED** [review]
Add periods to descs

**Review by Cyberboss — APPROVED** [review]

**Pull request merged**